include/fmt/format-inl.h

```diff
@@ -1790,8 +1790,8 @@ bool check_divisibility_and_divide_by_pow5(uint32_t& n) FMT_NOEXCEPT {
 template <int N> uint32_t small_division_by_pow10(uint32_t n) FMT_NOEXCEPT {
   static constexpr struct {
     uint32_t magic_number;
-    int shift_amount;
-    int divisor_times_10;
+    uint32_t shift_amount;
```

Comment:
Is this necessary? The warning only refers to `divisor_times_10`.

Reply:
Can `shift_amount` be less then 0?

Reply:
No but if this doesn't cause a warning, please keep it as is.

Reply:
Are there any guarantees that it won't give out in the future?